src/types/filterTypes.ts

```diff
@@ -0,0 +1,9 @@
+export type Filters = {
+  property_name: 'isin' | 'shortname';
+  values: string[];
+};
+
+export type AvailableFilterOptions = {
```

Comment:
Please update according to the new data type:
```
class Filters(BaseModel):
    isin: list[str]
    issuer_name: list[str]
    filename: list[str]
    industry: list[str]
    risk_type: list[str]
    green: list[str]
```
Notice: shortname is no longer present.